examples/README.md

```diff
@@ -1,22 +1,24 @@
 # Examples
 
 > [!WARNING]
-> The examples in this folder run against the `main` branch. There may be backwards
-> incompatible changes compared to the [latest stable version](https://github.com/ratatui-org/ratatui/releases/latest).
 >
-> If you find that code copied from an example fails to run in your own application, please check
-> the tag of the release that your project is using.
+> There are backwards incompatible changes in these examples, as they are designed to compile
+> against the `main` branch.
 >
-> To view examples that match the version of Ratatui that you are using checkout the matching tag in
-> your local git repository, or select the tag in the "Switch branches/tags" button if you're
-> viewing this file on GitHub.
+> There are a few workaround for this problem:
 >
-> To use incompatible example code as is, you can also use the most recent alpha release of Ratatui
-> (we release these weekly).
-
-These gifs were created using [VHS](https://github.com/charmbracelet/vhs). Each example has a
-corresponding `.tape` file that holds instructions for how to generate the images. Note that the
-images themselves are stored in a separate git branch to avoid bloating the main repository.
+> - View the examples as they were when the latest version was release by selecting the tag that
+>   matches that version. E.g. <https://github.com/ratatui-org/ratatui/tree/v0.25.0/examples>. There
+>   is a combo box at the top of this page which allows you to select any previous tagged version.
+> - To view the code locally, checkout the tag using `git switch --detach v0.25.0`.
```

Comment:
nit/question:

`git checkout <tag>` also work here and is more common.

I'd be interested to learn if there is actually any difference/reason to chose this over `checkout`?

Reply:
`git switch` is the newer command. It was created to have fewer edge cases to deal with and should generally be the command to recommend to most users. `checkout` does too many things, which makes it easy to do the wrong thing with the wrong combinations of parameters.

Reply:
https://github.com/git/git/blob/master/Documentation/RelNotes/2.23.0.txt#L61

Reply:
TIL; thanks!